Podfile.lock

```diff
@@ -1,30 +1,44 @@
 PODS:
-  - Alamofire (4.8.2)
-  - Japx (2.1.0):
-    - Japx/Core (= 2.1.0)
-  - Japx/Alamofire (2.1.0):
-    - Alamofire (~> 4.8)
+  - Alamofire (5.2.1)
+  - FlagPhoneNumber (0.8.0):
+    - FlagPhoneNumber/libPhoneNumberiOS (= 0.8.0)
+  - FlagPhoneNumber/libPhoneNumberiOS (0.8.0)
+  - Japx (3.0.0):
+    - Japx/Core (= 3.0.0)
+  - Japx/Alamofire (3.0.0):
+    - Alamofire (~> 5.0)
```

Comment:
Don't update dependencies which are not part of this PR/Fix
In general, don't change/make unrelated changes, keep it precise and to the point whenever you're working on an open-source project